pages/gm/f21.js

```diff
@@ -84,16 +110,74 @@ function F21() {
 				}}
 			/>
 			<Banner decorative />
+      <div className="text-center">
+        <img className="gm-graphic" src="/images/Fall_GM_2021_graphic.png" width="1000px" alt="Fall GM 2021 Marketing Graphic"/>
+      </div>
+      <Countdown date={GM_START_TIME} renderer={countdownRenderer}/>
 			<div className="content-container-tight text-center">
-				<h1>Fall &apos;21 General Meeting</h1>
-        <Link href={RSVP_LINK}>
-          <a className="button" target="_blank" rel="noreferrer noopener">
-            RSVP Now!
-          </a>
-        </Link>
-        <p>Check back later for more information on COVID safety, location, and the agenda!!</p>
+        <div id="info-wrapper">
+          <h2>Relevant information</h2>
+          <div className="flex">
+            <div>
+              <h3>How to get there</h3>
+              <img src="/images/Fall_GM_2021_temp_map.png" alt="temporary FPO image of google maps" width="400px"/>
+            </div>
+            <div className="what-to-bring">
+              <h3>What to bring</h3>
+              <p>Required: Face Mask, Bruincard</p>
+              <span>(learn more about how ACM is <Link href={"/covid"}><a target="_blank" rel="noreferrer noopener">committed to running COVID safe events</a></Link> in our return to campus)</span>
+              <p>Encouraged: Laptop or Phone to view slides, excitement to learn about ACM!</p>
+            </div>
+          </div>
+        </div>
+
+        <div id="gm-program-wrapper">
+          <h2>Program</h2>
+					<div className="gm-program-section">
+            <div className="section-header">
+              <h3>Welcome</h3>
+              <h4>[COPY CHECK] Welcome from president, Matt Wang</h4>
+            </div>
+					</div>
+          <div className="gm-program-section">
+            <div className="section-header">
+              <h3>Committee Presentations</h3>
+              <h4>[COPY CHECK] Learn what each of ACM&rsquo;s eight committees has in store for Fall quarter</h4>
+            </div>
+            <p className="gm-program-row"><img src="/images/committees/studio_wordmark.svg"/> Christian Loanzon, Peter Sutarjo</p>
```

Comment:
Re: these images; obv we'll have to disable some ESLint rules / add alt tags, but could we generate this entire thing from a `.map()` chain? I think it might be easier than hard-coding, but obv I know this is a bit quick & dirty.

Reply:
Could do it, but for the sake of a quick ship I'll probably keep hard coded for now?

I do have a good data structure in mind for generating this program though, might be a good new issue